ui/src/app/pages/data/concept/concept-homepage.spec.tsx

```diff
@@ -1,9 +1,9 @@
 import {mount} from 'enzyme';
 import * as React from 'react';
 
+import {ConceptHomepage} from 'app/pages/data/concept/concept-homepage';
```

Comment:
(Opt) Is there a particular reason for shifting this line up a few lines? If not, I'd leave as-is to distract less from meaningful changes.

Reply:
the linter wanted this to be alphabetized